### PR TITLE
data:Brownei

### PR DESCRIPTION
**Pull request by Brownei**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5205"></a>

### CODE REVIEW

File: data/Brownei.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+"name" : "Brownson Esiti",
+"type" : "personal",
```
Comment (SurajPratap10) [inline data/Brownei.json:3]: What's wrong with this and it's too big, brief in 1-2 lines.

**Comment by SurajPratap10** [comment]
What's wrong with the bio part it's too long, brief in 1-2 lines.

File: data/Brownei.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+"bio" :  "Frontend Web Developer who just started his career less than a year ago. I have always been fascinated with the power of the internet and how websites are built. I started learning HTML, CSS, and JavaScript on my own and quickly discovered my undying passion for frontend development. I am well-versed in a frontend framework which is React and I do hope to always stays up-to-date with the latest web technologies.
```
Comment (Sindhuinti) [inline data/Brownei.json:4]: Please make your bio to single link and place a comma `,` at the end.

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

**Comment by SaraJaoude** [comment]
> What's wrong with the bio part it's too long, brief in 1-2 lines.

@SurajPratap10 whilst there are advantages to having a short bio this is not a requirement and we encourage people to write the bios they want (and these can always be updated). Clearly there is nothing wrong with this bio -  please do not write such rude and unsupportive messages in the future, this is not collaborative or welcoming.

**Comment by SurajPratap10** [comment]
> > What's wrong with the bio part it's too long, brief in 1-2 lines.
> 
> @SurajPratap10 whilst there are advantages to having a short bio this is not a requirement and we encourage people to write the bios they want (and these can always be updated). Clearly there is nothing wrong with this bio - please do not write such rude and unsupportive messages in the future, this is not collaborative or welcoming.

It was too long that's why I told him to write a shorter bio as it gives greater engagement from a reader's POV. Sorry, if it looked rude will take care of it in the future,

**Comment by Brownei** [comment]
Thank you very much for the assistance

**Review by devvsakib — CHANGES_REQUESTED** [review]
Your bio is way too long, you can make it simple, in 1/2 line. add a comma here
image

**Comment by SurajPratap10** [comment]
> Your bio is way too long, you can make it simple, in 1/2 line. add a comma here
> image
> 

Ya, and as per your wish if you want to keep this bio then no worries but add the comma @Brownei  as this PR is opened from a long time.

**Comment by eddiejaoude** [comment]
@devvsakib please do not comment on people's bio's, it is **their** bio - also if you look further up the Pull request timeline you can see this has already been discussed

File: data/Brownei.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
+{
+"name" : "Brownson Esiti",
+"type" : "personal",
+"bio" :  "Frontend Web Developer who just started his career less than a year ago. I have always been fascinated with the power of the internet and how websites are built. I started learning HTML, CSS, and JavaScript on my own and quickly discovered my undying passion for frontend development. I am well-versed in a frontend framework which is React and I do hope to always stays up-to-date with the latest web technologies. 
+          Despite my short experience, I has already completed several projects, ranging from small static websites to large e-commerce platforms. I understand the importance of creating user-friendly and visually appealing interfaces and I do hope will strive to exceed the clients' expectations.
+          I is a team player and enjoys working collaboratively with other developers, designers, and project managers. I am always willing to share my knowledge and learn from my colleagues.
+          I is constantly seeking new challenges and opportunities to improve my skills and knowledge."
```
Comment (SaraJaoude) [inline data/Brownei.json:7]:
```suggestion
"bio" :  "Frontend Web Developer who just started his career less than a year ago. I have always been fascinated with the power of the internet and how websites are built. I started learning HTML, CSS, and JavaScript on my own and quickly discovered my undying passion for frontend development. I am well-versed in a frontend framework which is React and I do hope to always stays up-to-date with the latest web technologies. Despite my short experience, I has already completed several projects, ranging from small static websites to large e-commerce platforms. I understand the importance of creating user-friendly and visually appealing interfaces and I do hope will strive to exceed the clients' expectations. I is a team player and enjoys working collaboratively with other developers, designers, and project managers. I am always willing to share my knowledge and learn from my colleagues. I is constantly seeking new challenges and opportunities to improve my skills and knowledge."
```

File: data/Brownei.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
+{
+"name" : "Brownson Esiti",
+"type" : "personal",
+"bio" :  "Frontend Web Developer who just started his career less than a year ago. I have always been fascinated with the power of the internet and how websites are built. I started learning HTML, CSS, and JavaScript on my own and quickly discovered my undying passion for frontend development. I am well-versed in a frontend framework which is React and I do hope to always stays up-to-date with the latest web technologies. Despite my short experience, I has already completed several projects, ranging from small static websites to large e-commerce platforms. I understand the importance of creating user-friendly and visually appealing interfaces and I do hope will strive to exceed the clients' expectations. I is a team player and enjoys working collaboratively with other developers, designers, and project managers. I am always willing to share my knowledge and learn from my colleagues. I is constantly seeking new challenges and opportunities to improve my skills and knowledge."
```
Comment (SaraJaoude) [inline data/Brownei.json:4]:
```suggestion
"bio" :  "Frontend Web Developer who just started his career less than a year ago. I have always been fascinated with the power of the internet and how websites are built. I started learning HTML, CSS, and JavaScript on my own and quickly discovered my undying passion for frontend development. I am well-versed in a frontend framework which is React and I do hope to always stays up-to-date with the latest web technologies. Despite my short experience, I has already completed several projects, ranging from small static websites to large e-commerce platforms. I understand the importance of creating user-friendly and visually appealing interfaces and I do hope will strive to exceed the clients' expectations. I is a team player and enjoys working collaboratively with other developers, designers, and project managers. I am always willing to share my knowledge and learn from my colleagues. I is constantly seeking new challenges and opportunities to improve my skills and knowledge.",
```

**Comment by SaraJaoude** [comment]
Hi @Brownei I have fixed the json without changing any content from your bio (json does not accept line breaks/paragraphs) so I will now be merging your Profile

**Pull request merged**